src/BuildScriptGenerator/Node/NodeConstants.cs

```diff
@@ -34,6 +34,9 @@ public static class NodeConstants
         public const string YarnVersionCommand = "echo Using Yarn version: && yarn --version";
         public const string HugoVersionCommand = "echo Using Hugo version: && hugo version";
         public const string YarnPackageInstallCommand = "yarn install --prefer-offline";
+        public const string Yarn2PackageInstallCommand = "yarn plugin import workspace-tools && yarn workspaces focus --all";
+        public const string Yarn1ConfigFolderName = "cache-folder";
+        public const string Yarn2ConfigFolderName = "cacheFolder";
```

Comment:
is this someting yarn2 has recommened?

Reply:
https://next.yarnpkg.com/getting-started/migration Yes, on the bottom CLI command section, it migrated yarn install command into this command.

Reply:
@qianz2  not about yarn install ... looking at yarn cache config folder name. looks like yarn1 is cache-folder while yarn2 is cacheFolder. I'm curious why is that?

Reply:
Since you're passing the `--all` flag but not the `--production` flag as well it's the same as if you were to run `yarn install`

> looks like yarn1 is cache-folder while yarn2 is cacheFolder. I'm curious why is that?

We made the settings camelCase in v2